kvbc/src/kvbc_adapter/replica_adapter.cpp

```diff
@@ -0,0 +1,186 @@
+// Concord
+//
+// Copyright (c) 2022 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0
+// License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#include "kvbc_adapter/replica_adapter.hpp"
+#include "kvbc_adapter/blocks_deleter_adapter.hpp"
+#include "kvbc_adapter/kv_blockchain_adapter.hpp"
+#include "kvbc_adapter/app_state_adapter.hpp"
+#include "kvbc_adapter/state_snapshot_adapter.hpp"
+
+namespace concord::kvbc::adapter {
+
+ReplicaBlockchain::ReplicaBlockchain(
+    const std::shared_ptr<concord::storage::rocksdb::NativeClient> &native_client,
```

Comment:
I guess this is the source that should construct the blockchain and pass it to the lower level adapters

Reply:
And also the common boiler plate code as we discussed

Reply:
Thanks a lot @cloudnoize , you save me. This is really great comment, helped me a lot.